docs/build/guides/basics/classic-transition.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Shifting from Stellar Classic to smart contracts
+sidebar_position: 40
+description: Considerations for integrating with Stellar’s smart contracts specifically for wallets and exchanges
+---
+
+## 1. Shifting from Stellar Classic to Smart Contracts
```

Comment:
We can probably just remove this section title since it repeats the `title` attribute.